database/L/Little Simz/GREY Area/Venom

```diff
@@ -1,87 +1,79 @@
-Moving scatty all erratic and shit
-Niggas pussy licking batty and shit
-Oh you mad? Then come at me you prick
-Make a move better pattern it quick
-I assume you’ll be coming for blood
-That makes 2 of us that makes 2 of us
-Venom
+My, my, if you've ever heard what I've heard in the night
+What a fright, must've been a parasite in my past life
+I don't wanna ever come down from a high
+My soul ascended to the sky, it's just you and I
+God but I'm feelin' too alive
+Tryna get me out of spite
+Someone's gotta pay, I ain't talkin' big amounts
+Some kind of physical pain
+Some kind of traumatic shit
+Niggas wanna see dead bodies
+Probably not, they ain't rollin'
+With no automatic clips
+Moving scatty, all erratic and shit
+Niggas pussy lickin', batty and shit
+Oh, you mad? Then come at me, you prick
+Make a move, better pattern it quick
+I assume you'll be comin' for blood
+That makes two of us, that makes two of us
 
 Venom
 Venom
 Venom
+Venom, venom
 
 Night time to daylight we roll trees
-Saves live invade minds and go deep
-Oh he wouldn’t of guessed I’m so chill
-Part of the day in the life you won’t see
-Part of the day in the life you can’t be here for
-Its the day in the life of OG’s
-Dealing with inner demons that won’t leave
-No a word you will listen when my soul speaks
-Down I go
-So follow me follow me follow me
-Actually don’t follow me
-Nobody bother me
-Im a mess honestly
-Fuck all your policies
-They wanna kick me down a demolish me
-Use me and fucking abolish me
-I don’t wanna hear no apologies
+Saved lives, invade minds and go deep
+Oh, he wouldn't have guessed I'm so chill
+Part of the day in the life you won't see
+Part of the day in the life you can't be here for
+It's the day in the life of OG's
+Dealin' with inner demons that won't leave
+Not a word, you will listen when my soul speaks
+Down I go, so follow me, follow me, follow me
+Actually, don't follow me, nobody bother me
+I'm a mess, honestly, fuck all of your policies
+They wanna kick me down and demolish me
+Use me and fuckin' abolish me
+I don't wanna hear no apologies
 Or am I in my own head?
-Can’t tell anymore psychotic to the death
-When I dig deep I can never find nothing left
-Its a mystery
-Rage
-Nothing but rage
-Can’t figure out if I’m going insane
-Aint’ no doubt we’ll be showing no mercy so think twice if you wanna get blazed
-Nothing ain’t nice round here no games
-I don’t wanna dark nobody in a flash
-True its a stick up
-Hands in the air
-Put the money in bag all cash all cash
-Venom
+Can't tell anymore, psychotic to the death
+When I dig deep, I can never find nothin' left
+It's a mystery
+Rage, nothin' but rage
+Can't figure out if I'm goin' insane
+Ain't no doubt we'll be showing no mercy
+So think twice if you wanna get blazed
+Nothin' ain't nice 'round here, no games
+I don't wanna dark nobody in a flash
+True, it's a stick-up, hands in the air
+Put the money in the bag, all cash, all cash
 
 Venom
 Venom
+Venom
+Venom, venom
+
 Venom
```

Comment:
In the youtube video this line is absent, but if it's present on the album version I believe it'd be better to keep it.

Reply:
Yes, I've misplaced it I think. It's same on the album version too. Fixed on new commit